package.json

```diff
@@ -17,7 +17,7 @@
     "start": "lerna exec --parallel -- npm run start",
     "test": "cd library && npm test",
     "build": "npm run build:library && npm run build:webcomponent && npm run build:playground",
-    "build:library": "cd library && npm run prepare",
+    "build:library": "cd library && npm run prepack",
```

Comment:
what is wrong with `prepare`? why this has to change?

Reply:
`prepare` is run with `npm run bootstrap` which is `lerna bootstrap`. When you publish library from root `package.json` you execute `prepare` without `prepack` and we don't have Readme.md in npm https://www.npmjs.com/package/@asyncapi/react-component?activeTab=dependencies

Reply:
isn't it better to extend `prepare` than now maintain both, `prepare` and `prepack`, don't you think it will be confusing and lead to more problems?

Reply:
ok then.